### PR TITLE
[PAPI-238] Indexer OpenAPI examples

### PR DESCRIPTION
**Pull request by drmathias**
* OpenAPI examples
* Completed TODO for `GET /index/latest-block`

### CODE REVIEW

File: src/Opdex.Platform.Application/EntryHandlers/Blocks/GetLatestBlockQueryHandler.cs [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    {
+        private readonly IMapper _mapper;
+        private readonly IMediator _mediator;
+
+        public GetLatestBlockQueryHandler(IMapper mapper, IMediator mediator)
+        {
+            _mapper = mapper ?? throw new ArgumentNullException(nameof(mapper));
+            _mediator = mediator ?? throw new ArgumentNullException(nameof(mediator));
+        }
+        public async Task<BlockDto> Handle(GetLatestBlockQuery request, CancellationToken cancellationToken)
```
Comment (mrtpain) [inline src/Opdex.Platform.Application/EntryHandlers/Blocks/GetLatestBlockQueryHandler.cs:23]: nice

File: src/Opdex.Platform.WebApi/Controllers/IndexController.cs [hunk 2]
```diff
@@ -66,7 +77,7 @@ public async Task<IActionResult> ResyncFromDeployment(ResyncFromDeploymentReques
             var markets = await _mediator.Send(new RetrieveAllMarketsQuery(), cancellationToken);
             if (markets.Any())
             {
-                throw new Exception("Markets exist");
+                throw new NotFoundException("No markets exist to resync.");
```
Comment (mrtpain) [inline src/Opdex.Platform.WebApi/Controllers/IndexController.cs:80]: Consider changing this exception to explain that markets _do_ exist so their request is invalid. 

Maybe a bad request with "Markets already deployed" message.

**Review by mrtpain — APPROVED** [review]
one comment, LGTM, nice assist with the GetLatestBlock entry query

File: src/Opdex.Platform.WebApi/Controllers/IndexController.cs (hunk 2 above)
Comment (drmathias) [reply src/Opdex.Platform.WebApi/Controllers/IndexController.cs]: Sorted

**Pull request merged**